### ISSUE TITLE
Update call center documents and product video to reflect changes to 1095-B MVP design

### ISSUE THREAD

**Issue by tameracorson** [department-of-veterans-affairs/va.gov-team#38709]
- [x] Align on if update is in total of product (MVP + opt-in) or just opt-in; which items must be amended
- [ ] Iterate on call center documents using approved designs 
- [ ] Iterate on call center video script to reflect approved designs for opt-in

**Comment by tameracorson** [comment]
https://app.zenhub.com/workspaces/1095b-tax-form-61e19735167b2500170b2146/issues/department-of-veterans-affairs/va.gov-team/37014

**Comment by starlow00** [comment]
Hi @tameracorson. Is there an updated video available?

**Comment by CarlDickerson** [comment]
@starlow00 The product video has been revised and shared via Slack (the file was too large to upload here). I've also attached a link to download from box : https://mostudio.box.com/s/swnzeihhttb1np3n8umr89j4oz863wzz

**Comment by tameracorson** [comment]
@CarlDickerson @starlow00 Reopening 1095-B ticket r/t additional feature being added prior to launch and requirement for updated call center video to reflect the new design / feature / function.   the new feature is called opt-in for digital retrieval (aka opt-out of mail delivery)  This is a joint venture between 1095-B and VA profile team.

**Comment by starlow00** [comment]
@tameracorson Hello! Just checking in on this to see when an updated video might be available.

**Comment by tameracorson** [comment]
@starlow00 Still waiting on sitewide Content IA for input on designs.  They own the Drupal page that we use.

**Comment by tameracorson** [comment]
New ticket created
